composer.json

```diff
@@ -11,11 +11,11 @@
         }
     ],
     "require": {
-        "php": ">=7.2",
-        "firebase/php-jwt": "^5.0.0"
+        "php": "^8.0",
+        "firebase/php-jwt": "6.0.*"
```

Comment:
In this case I didn't want to go for the latest version since this repo is used by `public` and didn't want to force many changes in there, but I'll update it.